# Generated for NaoQi method ALVisionVideoInput.getAngPosFromImgPos
# Returns angles relative to camera axis given a normalized position in the image.

# normalized position in the image [0.0 - 1.0]
float32[] imgPos
---
# corresponding angles values in radians.
float32[] value


